doc_primer_orchestration.md

```diff
@@ -29,28 +30,30 @@ File structure
-* `build.py`: tooling for distributed build & SDFG load.
-* `dace_config.py`: DaCeConfig & DaCeOrchestration enum.
-* `orchestration.py`: main code, takes care of orchestration .scaffolding, build pipeline (including parsing) and execution.
-* `sdfg_opt_passes.py`: custom optimization pass for Pace, used in the build pipeline.
-* `utils.py`: as every "utils" or "misc" or "common" file, this should not exists and collect tools & functions I lazily didn't put in a proper place.
-* `wrapped_halo_exchange.py`: a callback-ready halo exchanger, which is our current solution for keeping the Halo Exchange in python (because of prior optimization) in orchestration.
+
+- `build.py`: tooling for distributed build & SDFG load.
+- `dace_config.py`: DaCeConfig & DaCeOrchestration enum.
+- `orchestration.py`: main code, takes care of orchestration .scaffolding, build pipeline (including parsing) and execution.
+- `sdfg_opt_passes.py`: custom optimization pass for Pace, used in the build pipeline.
+- `utils.py`: as every "utils" or "misc" or "common" file, this should not exists and collect tools & functions I lazily didn't put in a proper place.
+- `wrapped_halo_exchange.py`: a callback-ready halo exchanger, which is our current solution for keeping the Halo Exchange in python (because of prior optimization) in orchestration.
 
 DaCe Config
 -----------
 
 DaCe has many configuration options. When executing, it drops or reads a `dace.conf` to get/set options for execution. Because this is a performance-portable model and not a DaCe model, decision has been taken to freeze the options.
 
-`pace.dsl.dace.dace_config` carries a set of tested options for DaCe, with doc. It also takes care of removing the `dace.conf` that will be generated automatically when using DaCe. Documentation should be self-explanatory but a good one to remember is:
+`pace.dsl.dace.dace_config` carries a set of tested options for DaCe, with doc. It also takes care of removing the `dace.conf` that will be generated automatically when using DaCe.
 
-```python
-# Enable to debug GPU failures
-dace.config.Config.set("compiler", "cuda", "syncdebug", value=False)
-```
+Orchestration can be debugged by using the env var `PACE_DACE_DEBUG`.
 When set to `True`, this will drop a few checks:
-- `sdfg_nan_checker`, which drops a NaN check after _every_ computation on field _written_.
-- `negative_qtracers_checker` drops a check for `tracer < -1e8` for every written field named one of the tracers
-- `negative_delp_checker` drops a check for `delp < -1e8` for every written field named `delp*`
+
+- `sdfg_nan_checker`, which drops a NaN check after _every_ computation on field _written_,
+- `negative_qtracers_checker` drops a check for `tracer < -1e8` for every written field named one of the tracers,
+- `negative_delp_checker` drops a check for `delp < -1e8` for every written field named `delp*`,
+- `trace_all_outputs_at_index` drops a print on every variable at a given index to track numerical protection,
```

Comment:
It looks like you removed references to this particular check in favor of  _sdfg_nan_checker_ in the other changed files.  Were all references removed and if so, should this blurb be removed from the documentation?

Reply:
The check still exists and can be used. It is not deployed in the pipeline by default mostly because `trace_all_outputs_at_index` is very verbose. But there's cases where you need to track the numerics progression at a given index (for example to see how you get NaN down the line) where this is useful.